htdocs/lib/settings-dev.inc.php

```diff
@@ -105,7 +105,7 @@
 $sql_replacements['tmpdb'] = $tmpdbname;
 
 // safemode_zip-binary
-$safemode_zip = '/var/www/bin/phpzip.php';
+$safemode_zip = '/var/www/html/local/prodsys/phpzip.php';
```

Comment:
The purpose of `local/prodsys` so far is to contain files that must be copied to somewhere else upon installation. In production operation, the `local` dir is not there at all (it is deleted after checkout).

To stay with this convention, I suggest to move `phpzip.php` to `bin` directory of the OC code tree (/var/www/html/bin on the devel VM). Actually it would fit best to `htdocs/lib`, but I would not like to have it there for security reasons.